src/components/SafeListSidebar/SafeList/index.tsx

```diff
@@ -129,16 +136,19 @@ export const SafeList = ({ onSafeClick }: Props): ReactElement => {
                     }
                     defaultExpanded={shouldExpandOwnedSafes}
                   >
-                    {ownedSafesOnNetwork.map((address) => (
-                      <SafeListItem
-                        key={address}
-                        address={address}
-                        networkId={id}
-                        onSafeClick={onSafeClick}
-                        loadedSafes={loadedSafes}
-                        shouldScrollToSafe={!isSafeAdded(loadedSafes, address)}
-                      />
-                    ))}
+                    {ownedSafesOnNetwork.map((address) => {
+                      const isAdded = isSafeAdded(loadedSafes, address)
+                      return (
+                        <SafeListItem
+                          key={address}
+                          address={address}
+                          networkId={id}
+                          onSafeClick={onSafeClick}
+                          showAddSafeLink={isCurrentNetwork && !isAdded}
```

Comment:
Safes that are not 'added' are not saved in the store and cannot therefore be accessed outside of the current network.

Reply:
`isCurrentNetwork` can be removed. You should be able to add a safe from any network.

Reply:
I've removed it, but the Safes-to-add won't appear unless you have opened a network with to-add Safes. They are not stored locally unless they are added.